android/gradle/dependencies.gradle

```diff
@@ -64,11 +65,12 @@ def build = [
     errorProneTestHelpers: "com.google.errorprone:error_prone_test_helpers:${versions.errorProne}",
     nullAway: 'com.uber.nullaway:nullaway:0.9.0',
     gradlePlugins: [
-        android: 'com.android.tools.build:gradle:3.6.2',
+        android: 'com.android.tools.build:gradle:4.1.3',
```

Comment:
we should probably not bump this in order to keep behavior closer to that of the internal monorepo